composer.json

```diff
@@ -88,8 +91,15 @@
         "ext-phpiredis": "*"
     },
     "config": {
+        "optimize-autoloader": true,
         "platform": {
-            "php": "8.0"
+            "php": "8.0",
+            "ext-imagick": "3.7.0",
+            "ext-yaml": "2.2.2",
+            "ext-redis": "5.3.7",
+            "ext-swoole": "4.8.10",
+            "ext-zstd": "0.11.0",
+            "ext-mongodb": "1.13.0"
```

Comment:
I think it might be fair to say that we should probably use a complete wildcard or only the major version.

`"ext-imagick": "3.*",` or `"ext-imagick": "*",`

Reply:
Unfortunately, we can't:

```
$ composer update
Loading composer repositories with package information

                                
  [UnexpectedValueException]    
  Invalid version string "3.*"  
                                
```

Reply:
bummer, what about `^3`? 😄

Reply:
same:

```
$ docker run --rm --interactive --tty \
  --volume $PWD:/app \
  composer update
Loading composer repositories with package information

In VersionParser.php line 186:
                               
  Invalid version string "^3"  
```

Reply:
How about using a fake 0.0.0 value when possible? ext-mongodb requires at least 1.8.1 so I put that.